include/libkernelflinger/log.h

```diff
@@ -38,9 +38,15 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE * efi_log_file;
+UINTN      KERNELFLINGER_LOG_OUTPUT;
+EFI_FILE * g_root;
 
 void log(const CHAR16 *fmt, ...);
 void vlog(const CHAR16 *fmt, va_list args);
+EFI_FILE * log_init (const CHAR16 *filename) ;
+void       log_end (EFI_FILE *file) ;
```

Comment:
**Coding style violation**: Alignment is wrong.  it should be one space only between void and the name of the function.

Reply:
Already modified.